.github/workflows/mvn-build.yml

```diff
@@ -13,11 +13,13 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2.3.4
```

Comment:
Are you sure this is a good idea? I would expect the GH Actions to be updated automatically?

Reply:
Just wanted to do a test. I should have done a Draft PR